### ISSUE TITLE
Crash with latest library 9.0.3 in Android 4.X

### ISSUE THREAD

**Issue by skinner2012** [Instabug/Instabug-Android#252]
### Steps to Reproduce the Problem
1. Have Android smartphone which has Android 4.X OS.
2. Have an app using Instabug 9.0.3 and make sure the app will initilaize Instabug during launching.
3. Launch the app

### Expected Behavior
Should not crash. Should be retro-compatible for Android 4.X.

### Actual Behavior
Crash.

And here is the logs

```
11-15 22:49:35.781 23073-23417/xxxxx E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #7
    Process: xxxxxx, PID: 23073
    java.lang.RuntimeException: An error occured while executing doInBackground()
        at android.os.AsyncTask$3.done(AsyncTask.java:300)
        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
        at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
        at java.util.concurrent.FutureTask.run(FutureTask.java:242)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
        at java.lang.Thread.run(Thread.java:841)
     Caused by: java.lang.IllegalAccessError: tried to access method androidx.core.app.JobIntentService.dequeueWork:()Landroidx/core/app/JobIntentService$GenericWorkItem; from class androidx.core.a.a
        at androidx.core.a.a.dequeueWork(InstabugBackgroundService.java:1)
        at androidx.core.app.JobIntentService$CommandProcessor.doInBackground(JobIntentService.java:390)
        at androidx.core.app.JobIntentService$CommandProcessor.doInBackground(JobIntentService.java:383)
        at android.os.AsyncTask$2.call(AsyncTask.java:288)
        at java.util.concurrent.FutureTask.run(FutureTask.java:237)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587) 
        at java.lang.Thread.run(Thread.java:841) 
```

### Instabug integration code
```
        new Instabug.Builder(this, BuildConfig.INSTABUG_API_KEY)
                .setInvocationEvents(InstabugInvocationEvent.SHAKE)
                .build();

        BugReporting.setShakingThreshold(800);
        Instabug.setColorTheme(InstabugColorTheme.InstabugColorThemeDark);
        Instabug.setWelcomeMessageState(WelcomeMessage.State.DISABLED);
```

### SDK Version
9.0.3

### Android Version
4.4.4

### Device Model
Samsung S4

**Comment by mzelzoghbi** [comment]
Hey @skinner2012 , Thanks for reporting this, can you share with me the androidX support libraries you are using?

Thank You

**Comment by skinner2012** [comment]
```
    implementation 'androidx.multidex:multidex:2.0.1'
    implementation 'androidx.appcompat:appcompat:1.1.0'
    implementation 'androidx.cardview:cardview:1.0.0'
    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
    implementation 'androidx.vectordrawable:vectordrawable:1.1.0'
    implementation 'androidx.browser:browser:1.0.0'
    implementation 'androidx.exifinterface:exifinterface:1.0.0'
    implementation 'androidx.recyclerview:recyclerview:1.0.0'
    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
    implementation 'androidx.preference:preference:1.1.0'
```

**Comment by SKerencic** [comment]
have the same issue...any updates on this? thank you

**Comment by mzelzoghbi** [comment]
Hey @skinner2012 @SKerencic  

We created a snapshot that should fix the issue, Can you please try it and let us know if everything works out?

```
implementation 'com.instabug.library:instabug: 9.0.3.5-SNAPSHOT'
```
And don't forget to add this to your app **build.gradle** before your dependencies
```
repositories {
    ....
    maven {
        url "https://oss.sonatype.org/content/repositories/snapshots"
    }
    ...
}
```
Thank You

**Comment by SKerencic** [comment]
@mzelzoghbi sorry for my late reply... this fix is working for me. tested on android 4.4.4.. great!

**Comment by KORuL** [comment]
unresolved reference

**Comment by mzelzoghbi** [comment]
@KORuL Can you elaborate more?

**Comment by KORuL** [comment]
Sorry? in `implementation 'com.instabug.library:instabug: 9.0.3.5-SNAPSHOT'` you added space

**Comment by mzelzoghbi** [comment]
@KORuL sorry my bad, I updated it

**Comment by SKerencic** [comment]
@mzelzoghbi when this fix will be released/merged into new version? thanks

**Comment by mzelzoghbi** [comment]
@SKerencic this will be included in our next release this is planned to be within 2 weeks.

**Comment by ThiagoPierotti** [comment]
Joining the discussion here - 9.0.3 still crashes with:

**Comment by mzelzoghbi** [comment]
Hi @ThiagoPierotti 
This should be fixed on our latest release 9.0.4. Please let us know if you faced any other issues. 

Thank You
